.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": [["@babel/preset-env", { "modules": false }]]
```

Comment:
Why are you going to use Babel?

Reply:
This is my mistake. I wanted to write a Babel plugin to replace paths. I forgot to delete it.